wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md

```diff
@@ -24,10 +24,10 @@ In stable, the accuracy (and judgement) requirements for each [grade](/wiki/Game
 | Grade | osu! / osu!taiko | osu!catch | osu!mania |
 | :-: | :-- | :-- | :-- |
 | SS | 100% | 100% | 100% |
-| S | ≥90% (≤1% MEHs/50s, no misses) | ≥98% | ≥95% |
-| A | ≥80% (no misses) or ≥90% | ≥94% | ≥90% |
-| B | ≥70% (no misses) or ≥80% | ≥90% | ≥80% |
-| C | ≥60% | ≥85% | ≥70% |
+| S | >90% GREATs/300s (≤1% 50s, no misses) | >98% | >95% |
```

Comment:
why was the "MEH" term removed here while "GREAT" was left?

Reply:
Fixed. I'm sorry for this. I forgot that MEH judgements don't exist on taiko and ended up misunderstanding the reasoning to include them here.